platform/mellanox/rules.mk

```diff
@@ -48,4 +48,7 @@ $(SYNCD)_RDEPENDS += $(MLNX_SAI)
 # Inject mlnx sdk libs to platform monitor
 $(DOCKER_PLATFORM_MONITOR)_DEPENDS += $(APPLIBS) $(SX_COMPLIB) $(SXD_LIBS) $(SX_GEN_UTILS) $(PYTHON_SDK_API) $(APPLIBS_DEV) $(SX_COMPLIB_DEV) $(SXD_LIBS_DEV) $(SX_GEN_UTILS_DEV)
 
+# Force the target bootloader for mellanox platforms to grub regardless of arch
+TARGET_BOOTLOADER = grub
```

Comment:
All mellanox devices use grub and afaik all planned devices will use grub as well. 

This is the mellanox platform rules file. 

In the future more logic could be added here but for now this is safe. 

I am forcing this here because the default logic I implemented is for amd64 devices to use grub and arm64 device to use uboot in order to minimize changes to other platforms code.